server/frontend/src/containers/Register/index.js

```diff
@@ -1,18 +1,193 @@
 // src/containers/Register/index.js
 
 import React, { Component } from 'react';
-import classnames from 'classnames';
+//import classnames from 'classnames';
+<head>
+  <script src="http://fb.me/react-0.10.0.min.js"></script>
+  <script src="http://fb.me/JSXTransformer-0.10.0.js"></script>
+</head>
 
+
+/*
 class Register extends Component {
   render () {
     const { className, ...props } = this.props;
 
     return (
       <div className={classnames('Register', className)} {...props}>
-        <h2>Hello from Register!</h2>
-      </div>
-    );
-  }
+            <h2>Hello from Register!</h2>
+	    </div>
+    )
+     
+    }
+   
+}
+*/
+//for the database info
+
+
+
+
+//registration form
+class Register extends Component {
+    constructor(props) {
+	super(props);
+	this.state = {
+	    fname: '',
+	    lname: '',
+	    NSHE:0,
+	    email: '',
+	    pnum:0,
+	    vColor:'',
+	    vLic:'',
+	    vMake:'',
+	    vModel:'',
+	    value: 'student'
+	};
+
+	this.handleChange = this.handleChange.bind(this);
+	this.handleSubmit = this.handleSubmit.bind(this);
+	this.handlechange2 = this.handleChange2.bind(this);
+    }
+
+    handleChange(event) {
+	const target = event.target;
+	const value = target.value
+	const name = target.name;
+	this.setState({
+	    [name]:value,
+	   
+	    
+	});
+    }
+    handleChange2(event){
```

Comment:
since we're not using `handleChange` remove it and rename this one to `handleChange`.